src/Theme/Theme.stories.tsx

```diff
@@ -12,32 +14,21 @@ export default {
 export const Default: Story<ThemeProps> = () => {
   return (
     <div className="flex flex-col gap-y-4">
-      <Theme dataTheme="dark" className="p-8 bg-neutral rounded-2xl">
-        <div className="flex gap-x-2">
-          <Button>Default</Button>
-          <Button color="primary">Primary</Button>
-          <Button color="secondary">Secondary</Button>
-          <Button color="accent">Accent</Button>
-        </div>
-      </Theme>
-
-      <Theme dataTheme="cupcake" className="p-8 bg-neutral rounded-2xl">
-        <div className="flex gap-x-2">
-          <Button>Default</Button>
-          <Button color="primary">Primary</Button>
-          <Button color="secondary">Secondary</Button>
-          <Button color="accent">Accent</Button>
-        </div>
-      </Theme>
-
-      <Theme dataTheme="lofi" className="p-8 bg-neutral rounded-2xl">
-        <div className="flex gap-x-2">
-          <Button>Default</Button>
-          <Button color="primary">Primary</Button>
-          <Button color="secondary">Secondary</Button>
-          <Button color="accent">Accent</Button>
-        </div>
-      </Theme>
+      {DEFAULT_THEMES.map((theme) => (
```

Comment:
Nice 😄

Reply:
TY, going to remove the test story and then merge